custom.Makefile

```diff
@@ -17,5 +24,10 @@ MONDO_JSON=http://purl.obolibrary.org/obo/mondo/mondo.json
 $(TMP_DIR)/mondo.json:
 	wget $(MONDO_JSON) -O $@
 
-$(MAPPINGS_DIR)/mondo_all.sssom.tsv: $(TMP_DIR)/mondo.json
-	sssom parse $< --input-format obographs-json -m config/metadata.yaml --prefix-map-mode merged -o $@
+.PHONY: mondo_all
+mondo_all: $(TMP_DIR)/mondo.json
+	sssom parse $< --input-format obographs-json -m config/metadata.yaml --prefix-map-mode merged -o $(MAPPINGS_DIR)/mondo_all.sssom.tsv
```

Comment:
This is a bit ideosyncractic for makefiles - we should only use phony goals to give names to specific pipelines. Why did you remove `$(MAPPINGS_DIR)/mondo_all.sssom.tsv`?

Reply:
oh i just forgot to switch it back, i think this was an attempt to fix an error in trying to run `make all`, I'm just not super fluent in a lot of the intricacies of makefiles